packages/devextreme-scss/scss/widgets/material/chat/_index.scss

```diff
@@ -48,4 +48,7 @@
   $chat-messagelist-empty-prompt-font-size,
   $chat-messagelist-empty-prompt-color,
 );
-@include chat-messagelist($chat-messagelist-padding);
+@include chat-messagelist(
+  $chat-messagelist-padding,
+  $chat-messagelist-empty-icon-color,
```

Comment:
Could you please create separate variable for date-header color? If it needs to be the same as the empty icon color, then this definition should be inside the color.scss file

Reply:
done